README.esp.md

```diff
@@ -4,22 +4,22 @@
 
 La [integración de Lua](https://www.youtube.com/watch?v=IP3J56sKtn0) como [lenguaje de primera clase 
 dentro de Neovim](https://github.com/neovim/neovim/wiki/FAQ#why-embed-lua-instead-of-x) se perfila como una de sus características más destacadas. Sin embargo, la cantidad 
-de material didactico para aprender a escribir plugins en Lua no es tan grande como la que se puede 
+de material didáctico para aprender a escribir plugins en Lua no es tan grande como la que se puede 
 encontrar para escribirlos en Vimscript. Esta guia es un intento de proveer información básica a 
 las personas para que puedan empezar.
 
-Esta guia asume que tu estás unando, al menos, la version 0.5 de Neovim.
+Esta guía asume que tú estás usando, al menos, la versión `0.5` de Neovim.
```

Comment:
Para hacerlo más natural sugiero elimitar los "tú" (**estás** ya hace referencia a tu por no que no es necesario hacerlo explicito).

Reply:
That makes sense, I also felt it strange. Will do. Thanks for the suggestion!